backend_module/Docker-compose.yml

```diff
@@ -1,9 +1,27 @@
-version: '3'
+version: '3.5'
 services:
-  spring-boot-app:
-    image: spring-boot-image
+  mysql:
+    image: mysql:8
+    restart: always
+    hostname: localhost
+    ports:
+      - 5432:5432
+    environment:
+      MYSQL_ROOT_PASSWORD: rootpass
+      MYSQL_USER: testuser
+      MYSQL_PASSWORD: testpass
+      MYSQL_DB: teste
+  server:
+    image: spring-mysql
```

Comment:
pls register your docker file in docker hub

Reply:
https://hub.docker.com/repository/docker/morozzyaka/spring-mysql